src/main/java/carriermanagementsystem/WcdmaCarrierBuilder.java

```diff
@@ -8,9 +8,11 @@
 import java.util.List;
 
 /**
- * This class is a child class to CarrierBuilder. It builds WCDMACarrier objects.
+ * This class is a child class to CarrierBuilder. It builds WCDMACarrier
+ * objects.
  */
 public class WcdmaCarrierBuilder implements CarrierBuilder {
+    private static final int WCDMA_RF_PORTS_NUMBER = 2;
```

Comment:
I appreciate you adding this variable to remove the magic number (just putting '2' in the code) from this class. Nice work.